java/mapping/supported.js

```diff
@@ -0,0 +1,26 @@
+const CHECK_SETTINGS_OPTIONS = [
+    'frames',
+    'region',
+    'scrollRootElement',
+    'ignoreRegions',
+    'floatingRegions',
+    'accessibilityRegions',
+    'layoutRegions',
+    'ignoreDisplacements',
+    'sendDom',
+    'matchLevel',
+    'name',
+    'layoutBreakpoints',
+    'isFully',
+    'visualGridOptions',
+    'variationGroupId',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    "variationGroupId",
```